tests/ecs6-class/line.test.js

```diff
@@ -0,0 +1,128 @@
+const Line = require('../../modules/ecs6-class/line')
+const line = require('../../modules/ecs6-class/line')
+const point = require('../../modules/ecs6-class/point')
+
+const point1 = new point({ x: 1, y: 1 })
+const point2 = new point({ x: 2, y: 2 })
+const line1 = new line({ point1, point2, n: 2, slope: 2 })
+const TLine = new line({})
+
```

Comment:
don't declare on top of the module a list of parameter, each test should have its own. And if you need the same parameters for a group of tests, declare them inside the describe block in the `beforeAll(()=> { } )` function